[Security][Cases][9.3 & Serverless]: Search bar in case attachment tab (#4367)
elastic/docs-content: solutions/_snippets/add-case-observables.md

An observable is a piece of information about an investigation, for example, a suspicious URL or a file hash. Use observables to identify correlated events and better understand the severity and scope of a case. 

> To view and manage observables, go to the **Observables** tab. You can find the tab in the following places:

Reviewer: ```suggestion
View and manage observables from the **Observables** tab. You can find the tab in the following places:
```
